layers/+tools/lsp/funcs.el

```diff
@@ -1,4 +1,4 @@
-;;; packages.el --- Language Server Protocol functions File for Spacemacs
+;;; funcs.el --- lsp Layer funcs File for Spacemacs
```

Comment:
Please revert -- keep the change to `funcs.el` -- but Language Server Protocol should be fully spelled out.